recipe-server/client/control_new/utils/apiFetch.js

```diff
@@ -62,9 +62,9 @@ export default async function apiFetch(url, options = {}) {
     throw new ApiError(message, data);
   }
 
-  if (response.status !== 204) {
+  try {
     return response.json();
```

Comment:
@mythmon yeah that's what I was doing but then I realized I think this should go back to checking status code for 204 or actually error out if there is an issue with the JSON. 204's are the only time I expect this to error out. Anything else is probably an actual issue.